src/mesh/tools.rs

```diff
@@ -534,38 +531,38 @@ mod tests {
         Mesh::from_triangle_faces_with_vertices_and_normals(faces, vertices, vertex_normals)
     }
 
-    pub fn open_box_sharp_mesh(position: [f32; 3], scale: f32) -> Mesh {
+    pub fn open_box_sharp_mesh() -> Mesh {
         let vertex_positions = vec![
             // back
-            v(-1.0, 1.0, -1.0, position, scale), //0
```

Comment:
I believe you can now delete the `v` function as this was it's last use. You can check for this yourself by running `cargo check --tests`. (`cargo clippy --tests` works also, but there are so many errors there that you probably don't want to see it... or maybe you should :smile: )

Reply:
good!
so should I see the clippy or not?

Reply:
Well you probably should see it, but don't fix any of the warnings/errors yet. We will probably only fix the clippy errors at some point. Having clippy warnings in tests is fine.